src/pages/Intents/intentsDatastoreQuery.js

```diff
@@ -0,0 +1,76 @@
+import { useDataMutation, useDataQuery } from '@dhis2/app-runtime'
+import isEmpty from 'lodash/isEmpty'
+import { useEffect, useState } from 'react'
+import { CUSTOM_INTENTS, NAMESPACE } from '../../constants/data-store'
+import { createKeyCustomIntent, updateSharingMutation } from '../../modules'
+
+/**
+ * update data store
+ * key: Custom intents
+ * */
+
+export const saveCustomIntentsMutation = {
+    resource: `dataStore/${NAMESPACE}/${CUSTOM_INTENTS}`,
+    type: 'update',
+    data: ({ settings }) => ({
+        ...settings,
+        lastUpdated: new Date().toJSON(),
```

Comment:
this should work in the sense of getting ISO/UTC time, but I guess is inconsistent with the general practice of DHIS2 of recording times as server time zones, e.g. if my server is in Oslo, my update for now (13 March 14:42), would be recorded as `2025-03-13T14:42:00.000`, rather than as `2025-03-13T13:42:00.000Z`